app/Http/Requests/UpdateAttributes.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Traits\ApiRequest;
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateAttributes extends FormRequest
```

Comment:
Feel like 'SaveAttributes' would be a better name, as this also triggers when creating attributes for the first time?